http-server-netty/src/test/groovy/io/micronaut/http/server/netty/filters/FilterFormSpec.groovy

```diff
@@ -0,0 +1,55 @@
+package io.micronaut.http.server.netty.filters
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Requires
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.annotation.RequestFilter
+import io.micronaut.http.annotation.ServerFilter
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.multipart.MultipartBody
+import io.micronaut.runtime.server.EmbeddedServer
+import io.micronaut.scheduling.TaskExecutors
+import io.micronaut.scheduling.annotation.ExecuteOn
+import spock.lang.Issue
+import spock.lang.Specification
+
+class FilterFormSpec extends Specification {
+    @Issue('https://github.com/micronaut-projects/micronaut-core/issues/9917')
+    def "exception in form parameter"() {
+        given:
+        def ctx = ApplicationContext.run(["spec.name": "FilterFormSpec"])
```

Comment:
Can you replace: 

```groovy
     def ctx = ApplicationContext.run(["spec.name": "FilterFormSpec"])
        def server = ctx.getBean(EmbeddedServer)
        server.start()
```

with: 

```groovy
        EmbeddedServer server = ApplicationContext.run(EmbeddedServer, ["spec.name": "FilterFormSpec"])        
```

Reply:
i need the context to create the client though

Reply:
you can get the context from `server.getApplicationContext()`

Reply:
feels like the wrong way around of doing things to me

Reply:
We do ` EmbeddedServer server = ApplicationContext.run(EmbeddedServer,` a lot of times in our codebase.

Reply:
yea and i dislike it every time :)

Reply:
we do both approaches a lot.

Reply:
It is hard for me to understand why you don't want to replace three lines with one line. Do you as you want.